### ISSUE TITLE
Note that curl has to be installed on the system

### ISSUE THREAD

**Issue by joshuataylor** [appsignal/appsignal-elixir#222]
Alternatively, as appsignal-elixir relies on httpoison, why not just use httpoison?

**Comment by tombruijn** [comment]
Hi @joshuataylor, yes we should mention that in the docs here: http://docs.appsignal.com/elixir/installation.html
Going update that now and close the issue.

About HTTPoison, that was added quite recently for a support feature #199, and was not a core part of the package when we used curl to download the agent. HTTPoison is one of the things we like to remove as well in #140. We'll see if we can drop the curl dependency when using httpc.

Thanks for the report :)
